WorkspaceServer/Packaging/NuGetPackageExtensionLoader.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Try.Markdown;
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+using System.Threading.Tasks;
+
+namespace WorkspaceServer.Packaging
+{
+    public class NuGetPackageExtensionLoader
+    {
+        public NuGetPackageExtensionLoader()
+        {
+        }
+
+        public async Task LoadExtensionsInDirectoryAsync(FileSystemDirectoryAccessor rootDirectory, IKernel kernel)
```

Comment:
This should take `IDirectoryAccessor`.

Reply:
Split the thing into two, one which takes a `IDirectoryAccessor` and looks for dlls inside the extensions folder, and one which takes a assemblypath and loads the assembly present at that path